Assets/ScaleMonk Ads/Source/IAdsBinding.cs

```diff
@@ -20,7 +21,9 @@ public interface IAdsBinding
         bool IsRewardedVideoReadyToShow(string tag);
         bool AreRewardedEnabled();
         bool AreInterstitialsEnabled();
+        [Obsolete]
```

Comment:
Can we add a message suggesting that they should use the new method (with GDPRConsent as argument)